perl/p5-net-apns-persistent/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+perl5.branches      5.26 5.28
```

Comment:
5.30 was recently added to all existing module ports, so you should probably do it here too.

Reply:
I'm nor tracking: where should one use `${perl5.major}`, and where should one hardcode things like `5.30`?

Reply:
List the subports you want to define explicitly in perl5.branches. Within any of those subports, use ${perl5.major} which will be different depending on which subport you're in.